eng/ci/templates/build.yml

```diff
@@ -0,0 +1,39 @@
+jobs:
+  - job:
+    templateContext:
+      outputs:
+        - output: nuget
+          packagesToPush: "$(Build.ArtifactStagingDirectory)/*.nupkg"
+          packageParentPath: "$(Build.ArtifactStagingDirectory)"
+          nuGetFeedType: internal
+          publishVstsFeed: "e6a70c92-4128-439f-8012-382fe78d6396/c0493cce-bc63-4e11-9fc9-e7c45291f151"
+          # sbomBuildDropPath: '$(System.DefaultWorkingDirectory)/package'
```

Comment:
If we do not need this, please remove them.

Reply:
Done